### ISSUE TITLE
classDef / class not working with htmlLabels?

### ISSUE THREAD

**Issue by Daijobou** [knsv/mermaid#210]
With HTML in node this code not add class to node:

~~~~~
    graph LR;
        A-->B[AAA<span>BBB</span>];
        B-->D;
        classDef test fill:#FF0000,stroke:#FFFF00,stroke-width:4px;
        class A test;
~~~~~

Remove 
~~~~~
<span>BBB</span>
~~~~~
in "B" and its working fine. Is this a bug or my mistake?

**Comment by knsv** [comment]
I can reproduce this. Will dig further. Thinking the styling maybe should move out to css in the long run.

test could already be defined as below in a local stylesheet or similar:
```
        g .test rect{
            fill:#FF0000;
            stroke-width:4px;
        }
        g .test text{
            fill: #FFFF00 !important;
        }
```

```
    graph LR;
        A-->B[AAA<span>BBB</span>];
        B-->D;
        class A test;
```

**Comment by knsv** [comment]
Found the issue and fixed it. Also added some documentation on how to predefine classes in the css and set them from the graph definition.

This can be viewed here under the heading Css classes:
http://knsv.github.io/mermaid/#styling-and-classes

**Comment by knsv** [comment]
Fix for this issue was released in mermaid 0.5.2. Will close this now, let me know if you issues still remain.
